common-util/graphql/queries.js

```diff
@@ -0,0 +1,11 @@
+import { gql } from "graphql-request";
+
+export const emissionsQuery = gql`
+  {
+    epoches(orderBy: startBlock) {
```

Comment:
this plural was automatically generated, unfortunately I'm unable to provide the correct one